server/start.go

```diff
@@ -311,7 +314,7 @@ func startInProcess(ctx *server.Context, clientCtx client.Context, appCreator ty
 		_, port, _ := net.SplitHostPort(config.EVMRPC.RPCAddress)
 
 		// allocate separate WS connection to Tendermint
-		tmWsClient = ConnectTmWS(tmRPCAddr, tmEndpoint)
+		tmWsClient = ConnectTmWS(tmRPCAddr, tmEndpoint, ctx.Logger)
```

Comment:
Can we use the local client like the node itself, do we rely on specific feature of websocket?

Reply:
I don't understand the suggestion

Reply:
https://github.com/tharsis/ethermint/blob/main/server/start.go#L312

the api server already have a "local" tendermint rpc client, I wonder if we can just use that, not relevant to this PR though.

Reply:
This is one issue with current ws client though, if don't work when there's http_proxy env variables, maybe because the `tmRPCAddr` is `0.0.0.0:xxx`, rather than `127.0.0.1`.

Reply:
I see, yeah we can tackle that on a separate PR